mono/metadata/marshal.c

```diff
@@ -5001,7 +5001,7 @@ ves_icall_System_Runtime_InteropServices_Marshal_copy_to_unmanaged (MonoArrayHan
 	guint32 gchandle = 0;
 	gsize const bytes = copy_managed_common (src, dest, start_index, length, (gpointer*)&managed_source_addr, &gchandle, error);
 	if (bytes)
-		memcpy (dest, managed_source_addr, bytes); // no references should be involved
+		memmove (dest, managed_source_addr, bytes); // no references should be involved
```

Comment:
On Windows, memcpy and memmove are the same. I'm aware of the undefined-ness and a few instructions saved by memcpy in other versions, by not checking for overlap.

I suggest the memory could possibly maybe overlap, and better safe than sorry.
It is the slightest extra safety/correct behavior, in the most unsafe of places.